### DISCUSSION TITLE
Support for external frontends?

### DISCUSSION THREAD

**Discussion by aolko** [civitai/civitai#199] [Ideas]
Will you support external front-ends and/or back-ends utilizing civitai api? Since some people might rather implement lacking functionality themselves instead of waiting for official implementation.

**Comment by JustMaier** [comment]
We've got a [public API](https://github.com/civitai/civitai/wiki/REST-API-Reference) that can be used by external front-ends 👍
We haven't had the time to build the authenticated API yet, so that means that we don't have a way to support uploading, reviewing, etc yet.
